src/constants.js

```diff
@@ -59,6 +59,9 @@ export const CONFIG_DIRECTORY = getDirectory('config');
 export const LINK_REGISTRY_DIRECTORY = path.join(CONFIG_DIRECTORY, 'link');
 export const GLOBAL_MODULE_DIRECTORY = path.join(CONFIG_DIRECTORY, 'global');
 
+export const POSIX_GLOBAL_PREFIX = '/usr/local';
```

Comment:
From what I understand from NPM docs, we should to this in the opposite order: try `node`s folder first. That said not sure how well it would play with homebrew or my local folder hacks (I alias `node` to `~/Desktop/node8` etc. for quick testing purposes and I heard this is not that uncommon)